The api-doc endpoint is only available when there is at least one service (#3409)
mendix/docs: content/refguide/runtime.md

You can extend the functionality of the Runtime Server by writing Java actions. For more information,  see the [Runtime API](/apidocs-mxsdk/apidocs/#runtime) section of *API Documentation*.

{{% alert type="info" %}}

> If the app contains published services, links to available API documentation such as [OpenAPI documentation](open-api) for [published REST services](published-rest-services), links to [published OData services](published-odata-services), and WSDLs for [published web services](published-web-services), are available on the URL path `/api-doc` (for example: `https://myapp.mendixcloud.com/api-doc/`).

Reviewer: ```suggestion
If the app contains published services, links to available API documentation such as [OpenAPI documentation](open-api) for [published REST services](published-rest-services), links to [published OData services](published-odata-services), and WSDLs for [published web services](published-web-services), are available at [API Documentation](/apidocs-mxsdk/apidocs/).
```

Reviewer: The link that was inserted as an example does not exist, and if I understand this correctly, then users should be referred to the API docs which lead them to the relevant APIs

Reviewer 2: No,
We are talking about the api documentation for the service published by the app.
The user of the api (which might be Data Hub, for example) can read the documentation of the api by going to the URL `{applicationURL}/api-doc/` 

This will only exist for a valid application URL, but this is an example URL (myapp.mendixcloud.com)

Reviewer: Thanks - I missed the "for example"
```suggestion
If the app contains published services, links to available API documentation such as [OpenAPI documentation](open-api) for [published REST services](published-rest-services), [published OData services](published-odata-services), and WSDLs for [published web services](published-web-services), are available on the URL path `/api-doc` (for example: `https://myapp.mendixcloud.com/api-doc/`).
```

Reviewer: The suggestion looks good to me!
